src/lib_ccaligner/commons.cpp

```diff
@@ -5,6 +5,8 @@
 */
 
 #include "commons.h"
+#include <string>
+using namespace std;
```

Comment:
`using namespace std;` is considered bad practice. Please revert all changes related to this.